frontend/src/pages/post/Search/SearchPage.jsx

```diff
@@ -0,0 +1,26 @@
+import { useParams } from "react-router-dom"
+import { useState } from 'react';
+import Post from '../../../components/posts/post/PostController'
+import TextField from '@mui/material/InputBase';
+
+const SearchPage = () => {
+    const [searchTerm, setSeachTerm] = useState('')
+
+    return (
+        <div className="Search">
+            <TextField id="standard-basic" label="Standard" variant="standard" size="normal" placeholder="Seach..." onChange={event => {setSeachTerm(event.target.value);}} />
+            <br></br>
+            {Post.filter((val) => {
+            if (searchTerm=="") {
+            return val
```

Comment:
You can't use filter on an imported component, I am not sure what you are trying to do here. The page will not run

Reply:
My bad I assumed Post is structured the same as the mockdata so I used the same filter. Mind giving me a tip on how to get to the post contents(text) thank you.